docker-for-windows/networking.md

```diff
@@ -5,15 +5,15 @@ title: Networking features in Docker Desktop for Windows
 ---
 {% assign Arch = 'Windows' %}
```

Comment:
nit: is this still used now?

Reply:
Yes, this is still used in the networking topic.